ERP-BACKEND/app/routers/auth.py

```
@@ -91,7 +91,7 @@ def login(form_data: OAuth2PasswordRequestForm = Depends(), db: Session = Depend
         # Generic error message to prevent user enumeration
         raise HTTPException(status_code=401, detail="Invalid credentials")
 
-    user.last_login = datetime.utcnow()
+    user.last_login = datetime.now(timezone.utc)
```

Ruff (0.16.1)

[error] Undefined name `timezone`

(F821)